packages/prisma/src/prisma/migrations/20240810201009_basic_daily_analytics/migration.sql

```diff
@@ -0,0 +1,33 @@
+/*
+  Warnings:
+
+  - Made the column `color` on table `Course` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `order` on table `SessionBlock` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "Course" ALTER COLUMN "color" SET NOT NULL;
+
+-- AlterTable
+ALTER TABLE "SessionBlock" ALTER COLUMN "order" SET NOT NULL;
```

Comment:
We should remove this line and the one above from the migration, since they have been applied separately in the meantime and do not belong to this change

Reply:
fixed directly